Appraisals

```diff
@@ -285,6 +285,7 @@ elsif ruby_version?('2.2')
     gem 'makara', '< 0.5.0' # >= 0.5.0 contain Ruby 2.3+ syntax
     gem 'mongo', '>= 2.8.0'
     gem 'mysql2', '< 0.5'
+    gem 'pg'
```

Comment:
Can you add `>= 0.8.4` to the occurrences of 'pg' in `appraise 'contrib'` blocks, given that's our minimum version?

No need to touch `pg` under non-contrib blocks, as they are not related to your integration, and thus the code in this PR.